apps/reaper/lib/reaper/file_ingest/processor.ex

```diff
@@ -26,8 +27,13 @@ defmodule Reaper.FileIngest.Processor do
     dataset = Provisioner.provision(unprovisioned_dataset)
     filename = get_filename(dataset)
 
-    dataset
-    |> UrlBuilder.build()
+    http_extract_step =
```

Comment:
was also unsure if there will be geojson datasets that have more than one extract step, so I can change this function if needed. It's currently only looking at the last step, which it assumes should be an http step